healthz/utils/tls.go

```diff
@@ -52,6 +53,22 @@ func CertFromFilePair(certFile, keyFile string) (*tls.Certificate, error) {
 	return &cert, err
 }
 
+// AddCertToDefaultPool adds certificate to from file to default pool
```

Comment:
extra `to` in `adds certificate to from`
`AddCertToDefaultPool adds the certificate from file specified with fpath to default certificate pool`